psi4/driver/procrouting/proc.py

```diff
@@ -1518,7 +1518,7 @@ def scf_helper(name, post_scf=True, **kwargs):
         # PE needs information about molecule and basis set
         pol_embed_options = solvent.pol_embed.get_pe_options()
         core.print_out(""" Using potential file {} for Polarizable Embedding
-                       calculation.\n""".format(pol_embed_options.potfile))
+                       calculation.\n""".format(pol_embed_options["potfile"]))
```

Comment:
Would you mind changing this to an f-string?

Reply:
Done with pleasure 😄